server/models/project.js

```diff
@@ -38,6 +38,10 @@ const projectSchema = new Schema(
     serveSecure: { type: Boolean, default: false },
     files: { type: [fileSchema] },
     _id: { type: String, default: shortid.generate },
+    visibility: {
+      type: String,
+      enum: ['Private', 'Public']
```

Comment:
We want to add `{ default: 'Public' }`.  That won't update existing database records, but it does seem to add the property to the objects when they are returned from a query.